script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -671,12 +720,18 @@ It was fear, together with an inexpressible anger.
 [sha:1dab8ec8c38eddfa29c0c4aed2247ada6db6eae7]{
 -- Page 690, Offset 16520.
 -- “諦めていたからこそ許せたのだ”
-'It was because I had given up that I could overlook this.'
+// A little confused on this one. 許せた must have been something he did _before_ learning about this, no?
+// So it can't be that he forgave/overlooked this accumulation of knowledge.
+// Assuming it's more like he'd forgiven himself for giving up on his goal? Making peace with the impossibility of knowledge?
+'I had made peace with being unable to achieve my goal.'
```

Comment:
He had given up on knowing everything because he accepted that it was impossible, was my interpretation, though I did struggle with this line for a while.